packages/editor/src/client/panel/pageMetadataPanel.tsx

```diff
@@ -30,6 +30,11 @@ import {ImageSelectPanel} from './imageSelectPanel'
 import {ImageRefFragment} from '../api'
 
 import {useTranslation} from 'react-i18next'
+export interface PageMetadataProperty {
```

Comment:
Why did you add this?

Reply:
I merged changes in the master-branch into mine. This line was added during the merge I guess.

Reply:
Sorry. I got confused. I didn't know that github highlighted those changes.